README.md

```diff
@@ -17,58 +16,20 @@ su - stack
 git clone https://github.com/infrawatch/python-observabilityclient
 cd python-observabilityclient
 sudo python setup.py install --prefix=/usr
-
-# clone and install observability playbooks and roles
-git clone https://github.com/infrawatch/osp-observability-ansible
-sudo mkdir /usr/share/osp-observability
-sudo ln -s `pwd`/osp-observability-ansible/playbooks /usr/share/osp-observability/playbooks
-sudo ln -s `pwd`/osp-observability-ansible/roles/spawn_container /usr/share/ansible/roles/spawn_container
-sudo ln -s `pwd`/osp-observability-ansible/roles/osp_observability /usr/share/ansible/roles/osp_observability
-```
-
-### Enable collectd write_prometheus
-Create a THT environment file to enable the write_prometheus plugin for the collectd service. Then redeploy your overcloud and include this new file:
-
-```
-mkdir -p ~/templates/observability
-cat <EOF >> templates/observability/collectd-write-prometheus.yaml
-resource_registry:
-  OS::TripleO::Services::Collectd: /usr/share/openstack-tripleo-heat-templates/deployment/metrics/collectd-container-puppet.yaml
-
-# TEST
-# parameter_merge_strategies:
-#   CollectdExtraPlugins: merge
-
-parameter_defaults:
-  CollectdExtraPlugins:
-    - write_prometheus
-EOF
 ```
 
-### Discover endpoints
-After deployment of your cloud you can discover endpoints available for scraping:
-
-```
-source stackrc
-openstack observability discover --stack-name=standalone
-```
+## Usage
 
-### Deploy prometheus:
-Create a config file and run the setup command
+Use `openstack observabilityclient query somequery` to query for metrics in prometheus.
```

Comment:
Wouldnt it be better to reduce this to "openstack telemetry query somequery'? This way, we use telemetry consistently across the whole RHOSP18 instead of using in some places observability, in some places metrics, etc...

Reply:
I like the idea of naming the client "telemetry client" and unifying it with the telemetry operator.  There is a slight issue though, we decided to reuse this (python-observablilityclient) repository, because it already has some integration with RDO. Renaming it would mean quite a bit of added effort. What I could easily do is to just rename the command. If we decide, that a fact, that "python-observabilityclient" provides "openstack telemetry *" commands isn't confusing, then it's an easy change. We can discuss on our meeting today.